source/reference/command/planCacheSetFilter.txt

```diff
@@ -37,6 +37,16 @@ Definition
    do not persist after shutdown; however, you can also clear existing
    index filters using the :dbcommand:`planCacheClearFilters` command.
 
+   A sorting query on a :term:`sharded cluster` will always include the sort
+   key in each shard's list of :term:`projected <project>` fields. This allows
+   the :program:`mongos` instance to merge together the results from each shard
+   in sorted order, but also can result in a different query plan than if the
+   sort key were not projected.
+
+   To create an accurate index filter on a sharded cluster, make sure to
+   include any sorting key in the ``projection`` field. This ensures that the
+   index filter correctly models the query's behavior.
```

Comment:
There's a slight nuance that is not being caught here.  Let's refer back to the ticket and see if you can catch the slight but important nuance. 

Afterwards, let's have someone in engineering give a quick ok.  It should be relatively a quick check.